### PR TITLE
Scrum 119/integrate basic configuration parser

### PR DESCRIPTION
**Pull request by hiddensudo**

### CODE REVIEW

File: include/Server/include/Server.h [hunk 1]
```diff
@@ -19,17 +19,25 @@ namespace ISXSC {
 class SmtpServer {
 public:
     SmtpServer(boost::asio::io_context& io_context,
-               boost::asio::ssl::context& ssl_context, unsigned short port,
-               ThreadPool<>& thread_pool);
+               boost::asio::ssl::context& ssl_context);
     void Start();
 
 private:
+    std::string server_name;
```
Comment (shevmee) [inline include/Server/include/Server.h]: Needs refactor to meet code style

File: include/Server/src/Server.cpp [hunk 2]
```diff
@@ -1,50 +1,59 @@
...
+
 #include <array>
 #include <ctime>
 #include <fstream>
 #include <iostream>
-#include <sodium.h>
-#include "../../ErrorHandler/include/ErrorHandler.h"
 
+#include "../../ErrorHandler/include/ErrorHandler.h"
+#include "config.h"
```
Comment (shevmee) [inline include/Server/src/Server.cpp:11]: Include list needs refactor to meet code style(system libs, third praty libs, our libs)

**Review by shevmee — COMMENTED** [review]
The changes look promising and align well with the project's goals. The code is generally clean and well-structured. But pay attention on code convensions

**Comment by RomaB-5** [comment]
Parser Integration is done well.
Code style also seems to meet requirments after the last commit.
So, overall, the changes corresponds the targeted goal.

**Review by RomaB-5 — APPROVED** [review]

**Review by shevmee — APPROVED** [review]
Approved

**Pull request merged**